### ISSUE TITLE
Send desktop notifications to phone

### ISSUE THREAD

**Issue by AlexanderMB** [rootkiwi/an2linuxserver#3]
It would be nice to receive my libnotify notifications on my Android phone.

**Comment by rootkiwi** [comment]
Yes that's an idea, for now this app will focus on phone -> desktop. But possibly something that can be implemented in the future, maybe even a different app.
